src/ray/common/cgroup/cgroup_setup.cc

```diff
@@ -85,39 +90,27 @@ bool EnableCgroupSubtreeControl(const char *subtree_control_path) {
 
 namespace internal {
 
-bool CanCurrenUserWriteCgroupV2() { return access("/sys/fs/cgroup", W_OK | X_OK) == 0; }
+bool CanCurrenUserWriteCgroupV2() {
```

Comment:
why does this use the global `kCgroupDirectory` while the function below takes a specified path?

Reply:
Updated to take an argument.